### DISCUSSION TITLE
Firefly Docker - Export CSV wrong characters

### DISCUSSION THREAD

**Discussion by ISeeTWizard** [firefly-iii/firefly-iii#6166] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hi

My Firefly installation is in french and is docker based. (brand new installation)
I did an test export and I must see that characters like éàè aren't displayed correctly but like Ã‰vÃ¨nements

Can I change somewhere anything to avoid that? In the DB it's displayed correctly - Maybe not exported as UTF-8 or so?

Thanks for any idea - couldn't find anything yet

### Expected behaviour

It should export all the data also with special characters in it 

### Debug information

Debug information generated at 2022-06-14 10:44:33 UTC for Firefly III version **5.7.9**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.7.9 |
| Firefly III API | 1.5.6 |
| Build | 196, 09-06-2022 18:30:26 CEST |
| DB version | 18 (exp. 18) |
| PHP | `8.0.17` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ |  |
| Browser TZ | Europe/Paris |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | info, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| Trusted proxies (.env) | `**` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start | 2022-05-01 00:00:00 |
| Session end | 2022-05-31 23:59:59 |
| Session first | 2022-04-24 00:00:00 |
| User ID | 1 |
| User language | fr_FR |
| User locale | fr_FR |
| Attempt at "fr_FR.utf8" | 'fr_FR.utf8' |
| Attempt at "fr_FR.UTF-8" | 'fr_FR.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.63 Safari/537.36 Edg/102.0.1245.33 |

### Steps to reproduce

1. go to /export
2. click on button
3. open csv file within excel

**Comment by JC5** [comment]
I can't reproduce it, unfortunately. Make sure you import the file as UTF8?

**Comment by ISeeTWizard** [comment]
Hi
It's export, not import - here is the file exported
[2022_06_14_transaction_export.csv](https://github.com/firefly-iii/firefly-iii/files/8900049/2022_06_14_transaction_export.csv)
)

It's just a few entries with a category évènements - in the file you can see it...
Maybe I need an Environment Settings I forgot?

**Comment by JC5** [comment]
I know, but I assume you import the file in Excel?

<img width="598" alt="Screenshot 2022-06-14 at 18 54 15" src="https://user-images.githubusercontent.com/5889984/173633484-3a06375f-86a9-42b1-96fe-d49653ef6a39.png">

**Comment by ISeeTWizard** [comment]
Hi

thanks, I see now what you mean. Until now I could simply open every csv file and Excel recognized it correctly so I simply forgot the option to try with importing :/

When Importing and swapping to UTF-8 it works correctly

Thanks
